style.css

```diff
@@ -36,8 +31,9 @@ body {
   margin-left: 60px;
 }
 
-.headr .logo .menuBtn {
+.headr .logo .fa-bars {
   display: none;
+  font-size: 23px;
```

Comment:
- [Optional] Kindly, I think it would be better relative measurements like `em`, `rem` instead of absolute measurements, to make your content resizable 👌👍